src/Update/Updates.php

```diff
@@ -606,4 +606,43 @@ public function update_9001001() {
     $this->updater->getOutput()->writeln("");
   }
 
+  /**
+   * 9.3.0.
+   *
+   * @Update(
+   *    version = "9003000",
+   *    description = "Remove composer merge plugin."
+   * )
+   */
+  public function update_9003000() {
```

Comment:
If this does get put into a 10.0.x branch and released as 10.0.0, the annotation and method name will need to be updated.